tests/foreman/cli/test_reporttemplates.py

```diff
@@ -908,12 +908,14 @@ def test_positive_generate_hostpkgcompare(
             hosts_info.append(Host.info({'name': client.hostname}))
             client.enable_repo(REPOS['rhst7']['id'])
             client.install_katello_agent()
+            # Add ReX key
+            client.add_rex_key(satellite=default_sat)
         hosts_info.sort(key=lambda host: host['name'])
 
         host1, host2 = hosts_info
-        Host.package_install({'host-id': host1['id'], 'packages': FAKE_0_CUSTOM_PACKAGE_NAME})
-        Host.package_install({'host-id': host1['id'], 'packages': FAKE_1_CUSTOM_PACKAGE})
-        Host.package_install({'host-id': host2['id'], 'packages': FAKE_2_CUSTOM_PACKAGE})
+        Host.package_install(hosts=f"id={host1['id']}", package=FAKE_0_CUSTOM_PACKAGE_NAME)
+        Host.package_install(hosts=f"id={host1['id']}", package=FAKE_1_CUSTOM_PACKAGE)
+        Host.package_install(hosts=f"id={host2['id']}", package=FAKE_2_CUSTOM_PACKAGE)
```

Comment:
you could avoid Host.package_install altogether 
```suggestion
        res = host1.execute(f'yum -y install {FAKE_0_CUSTOM_PACKAGE_NAME} {FAKE_1_CUSTOM_PACKAGE}')
        assert not res.status
        res = host2.execute(f'yum -y install {FAKE_2_CUSTOM_PACKAGE}')
        assert not res.status
```

Reply:
Of course I can but for the sake of encapsulating this to a simple call, I think it's better to have a function for this than executing a call directly.

Reply:
Do you need to test that Satellite can install the packages when called through hammer? Or do you just need the packages installed?
If it is the latter, then directly installing is much simpler than all the layers that come with the old method.